pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -287,7 +287,44 @@ func describeProjectAndServer(project *projectapi.Project, server string) string
 	}
 }
 
-func describeDeploymentInServiceGroup(deploy graphview.DeploymentConfigPipeline) []string {
+func describeImageStreamTagRuntimeStatus(imageSpec string, c client.Interface, namespace string) string {
```

Comment:
@gabemontero you need to 1) traverse the graph instead of using a client directly and create a marker in pkg/build/graph/analysis that reports the issue reported by @deads2k and 2) [plug it in oc status](https://github.com/openshift/origin/blob/1d4d2070201bbb88f369d7661e1d489e2c016ea6/pkg/cmd/cli/describe/projectstatus.go#L252). See https://github.com/openshift/origin/pull/5029 for example.